internal/services/servicebus/servicebus_subscription_resource.go

```diff
@@ -111,6 +111,11 @@ func resourceServiceBusSubscription() *pluginsdk.Resource {
 				Optional: true,
 			},
 
+			"duplicate_detection_history_time_window": {
```

Comment:
Perhaps the default should be added in there. "The default value is 10 minutes".

Reply:
if that is the default value globally yes, otherwise it should be marked computed